mlflow/paddle/__init__.py

```diff
@@ -296,6 +303,7 @@ def load_model(model_uri, model=None, dst_path=None, **kwargs):
     import paddle
 
     local_model_path = _download_artifact_from_uri(artifact_uri=model_uri, output_path=dst_path)
+    pyfunc.utils._add_code_from_conf_to_system_path(local_model_path)
```

Comment:
Weird...is the `load_model` for this flavor in a different spot in the file than other flavors?

Reply:
what do you mean by a different spot? Isn't this still in the `load_model` function?

Reply:
I just meant literally in the order of the file. Only noticed cause you were modifying all the files

Reply:
ohhh yeah that is weird how it's located above `log_model`